patches/other-arch.diff

```diff
@@ -0,0 +1,186 @@
+Only in afl-other-arch: build.sh
+Only in afl: .gitignore
+diff -r afl/qemu_mode/build_qemu_support.sh afl-other-arch/qemu_mode/build_qemu_support.sh
```

Comment:
This seems to dumplicate `build_qemu_support.sh` above. Could you deduplicate (and check out the comments on the above script as well)?